src/hercule.coffee

```diff
@@ -38,6 +38,8 @@ transclude = (input, relativePath, parents, parentRefs, logger, cb) ->
 
     utils.inflate href, hrefType, (content) ->
       logger "Transcluding: #{href} (#{hrefType}) into #{parents[-1..][0]}"
+      if content is null && link.default
```

Comment:
Perhaps this should be moved up into a restructured logic block where `overridingReferences` are handled?